backend/calendar/to_do_list.py

```diff
@@ -34,8 +59,11 @@ def getList(self, usr_id):
 
         result = cursor.fetchall()
 
-        self.todo = result
-
-        cursor.close()
-        con.close()
+        if result[0] > 0:
+            to_do_list = result[0] 
```

Comment:
not sure how this works. Is result[0] all of the events or just the first one?
Need testing